Sources/Sentry/Public/SentryOptions.h

```diff
@@ -271,6 +271,14 @@ NS_SWIFT_NAME(Options)
  */
 @property (nonatomic, assign) BOOL enableSwizzling;
 
+/**
+ * This feature is experimental.
+ *
+ * When enabled, the SDK tracks performance for CoreData operations if auto
+ * performance tracking and enableSwizzling are enabled. The default is <code>NO</code>.
```

Comment:
```suggestion
 * When enabled, the SDK tracks the performance of Core Data operations. It requires enabling
 * performance monitoring. The default is <code>NO</code>.
 * @see <https://docs.sentry.io/platforms/apple/performance/>
```

Swizzling is enabled by default. We should educate users of what features will be lost if they flip that flag, on the documentation over that. This way we can avoid confusing the users with multiple options, particularly when they don't need to take any actions.